src/bin/vip-app-deploy.ts

```diff
@@ -252,27 +253,36 @@ Processing the file for deployment to your environment...
 const examples = [
 	// `app` subcommand
 	{
-		usage: 'vip app @mysite.develop deploy file.zip',
-		description: 'Deploy the given compressed file to your site',
+		usage: 'WPVIP_DEPLOY_TOKEN=1234 vip @example-app.develop app deploy file.zip',
+		description:
+			'Deploy a local archived file named "file.zip" that contains application code to a VIP Platform environment that has Custom Deployment enabled.',
 	},
 	{
-		usage: 'vip app @mysite.develop deploy file.zip --message "This is a deploy message"',
-		description: 'Deploy the given compressed file to your site',
+		usage:
+			'WPVIP_DEPLOY_TOKEN=1234 vip @example-app.develop app deploy file.tar --message="A description for this deploy"',
+		description:
+			'Add a descriptive message for the Custom Deployment of the archived file named "file.tar".',
```

Comment:
@rebeccahum does `vip app deploy` support plain `.tar` archives? Or do they have to be compressed (`.tar.gz`)?

Reply:
I suppose it doesn't? https://github.com/Automattic/vip-cli/blob/966c5e858b3ae357a667012b690a7657990add03/src/lib/validations/custom-deploy.ts#L18

```suggestion
		usage:
			'WPVIP_DEPLOY_TOKEN=1234 vip @example-app.develop app deploy file.tgz --message="A description for this deploy"',
		description:
			'Add a descriptive message for the Custom Deployment of the archived file named "file.tgz".',
```

Reply:
We do not support plain `.tar` archives. They have to be compressed.

Reply:
Thank you for catching this @sjinks & @rebeccahum ! ❤️